src/github.com/getlantern/balancer/balancer.go

```diff
@@ -163,9 +182,9 @@ func without(dialers []*dialer, i int) []*dialer {
 	}
 }
 
-// byQOSAscending implements sort.Interface for []*dialer based on the QOS
+// byQOSAscending implements sort.Interface for []*Dialer based on the QOS
 // (ascending)
-type byQOSAscending []*dialer
+type byQOSAscending []*Dialer
```

Comment:
This allows us to sort the `dialers` argument of `New()` instead of the former `dhs`.